tests/test_parser.py

```diff
@@ -395,7 +397,6 @@ def test_time_is_parsed(self, date_string, timeobj):
         param(date_string=u"11"),
         param(date_string=u"22:12:12 PM"),
         param(date_string=u"22:12:10:16"),
-        param(date_string=u"16:14 AM"),
```

Comment:
:lipstick: I would make this one of the passing tests as is, either in addition to the other two, or replacing one of them.